docs/documentation.md

```diff
@@ -46,46 +62,127 @@ All request should have in URL `stationId` param encoded with Base64.
     }
     ```
     
+   - Response
+
+   ```json
+    {
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "Task Two",
+      "description": "This is the second task",
+      "priority": "MEDIUM",
+      "isDone": False,
+      "date": "2023-11-15",
+      "notificationTime": "10:00:00"
+    }
+    ``
+    
 - `PATCH /tasks/{taskId}` - Update a specific task.
```

Comment:
I see that we have two PATCHes in this documentation. Duplication could be removed